### PR TITLE
Genres

### PR DESCRIPTION
**Pull request by CraigFerreira**

### CODE REVIEW

File: bin/seed.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const {
 
 const { users } = require('./data/users');
 const { concerts } = require('./data/concerts');
+const {genres}= require('./data/genres')
```
Comment (alejvillanueva) [inline bin/seed.js]: We can fix this easily but make sure you are following small linting/formatting 

 const { genres }= require('./data/genres')

File: server/api/genre.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// const qs = require('qs');
+const {Genre} = require('../db/index')
```
Comment (alejvillanueva) [inline server/api/genre.js]: Model should be imported after package dependencies and again spacing w/ brackets

**Review by alejvillanueva — CHANGES_REQUESTED** [review]
Can you make sure the formatting with spaces, semi-colons, etc. match what the rest of the code is doing since your linter isn't working

File: server/api/genre.js (hunk 2 above)
Comment (vikkiwchan) [reply server/api/genre.js]: Remove line 1, we don't need qs here.

File: server/api/genre.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const router = require('express').Router();
+const axios = require('axios');
+
+// GET api/genre
+router.get('/', async(req, res, next) => {
+    try{
+        let genreList= await Genre.findAll()
```
Comment (alejvillanueva) [inline server/api/genre.js]: space between genreList and =

**Review by alejvillanueva — APPROVED** [review]

**Pull request merged**